src/ImageListPage.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import ComponentPage from './ComponentPage.js';
+import HeaderBar from './HeaderBar.js';
+import {imagePath} from './constants';
+
+import './styles/ImageListPage.scss';
+
+function ImageListPage() {
+  return (
+    <div>
+      <HeaderBar />
+      <ComponentPage
+        hero={<ImageListHero />}
+        title='Image List'
+        description='Image lists display a collection of images in an organized grid.'
+        designLink='https://material.io/go/design-image-list'
+        docsLink='https://material.io/components/web/catalog/image-lists/'
+        sourceLink='https://github.com/material-components/material-components-web/tree/master/packages/mdc-image-list'
+        demos={<ImageListDemos />}
+      />
+    </div>
+  );
+}
+
+function ImageListHero() {
+  const items = [];
+  for (let i = 0; i < 15; i++) {
+    items.push({});
+  }
+
+  return (
+    <div>
+      <ImageList className='hero-image-list' items={items} includeAspectContainer />
+    </div>
+  );
+}
+
+function ImageListDemos() {
+  const standardImageListItems = [];
+  for (let i = 1; i <= 15; i++) {
+    standardImageListItems.push({
+      src: `${imagePath}/photos/3x2/${i}.jpg`,
+      label: 'Text label'
+    });
+  }
+
+  const masonryImageListItems = [
+    {
+      src: `${imagePath}/photos/3x2/16.jpg`,
+      label: 'Text label',
```

Comment:
can we get rid of the label property here? Its the same in every object, so just hard code it. Then all these 2 arrays of objects can just be 2 arrays of strings. 

Also seems like we could inline `${imagePath}/photos` in the images. 
```
<ImageListItem src={`${imagePath}/photos/${src}`} ... />
```

The arrays would then look like:

```
[ '3x2/16.jpg', '2x3/1.jpg, ... ]
```